docs/t-sql/statements/backup-transact-sql.md

```diff
@@ -325,6 +325,8 @@ If you choose to encrypt you will also have to specify the encryptor using the e
 - SERVER CERTIFICATE = Encryptor_Name
 - SERVER ASYMMETRIC KEY = Encryptor_Name
 
+The SERVER CERTIFICATE and SERVERY ASSYMETRIC KEY are a certificate and an assymetric key created in the Master database. For more informaiton see [CREATE CERTIFICATE](../../t-sql/statements/create-certificate-transact-sql.md) and [CREATE ASYMMETRIC KEY](../../t-sql/statements/create-asymmetric-key-transact-sql.md) respectively. 

```

Comment:
Several grammatical issues should be fixed before merging this. The fixed version is:

The SERVER CERTIFICATE and SERVER ASYMMETRIC KEY are a certificate and an asymmetric key created in the `master` database. For more information see [CREATE CERTIFICATE](../../t-sql/statements/create-certificate-transact-sql.md) and [CREATE ASYMMETRIC KEY](../../t-sql/statements/create-asymmetric-key-transact-sql.md), respectively.